js/site.js

```diff
@@ -59,267 +75,233 @@ if (location.hash) location.hash = '';
+  if ($("#social").val()) note_body += "⚠️ contact:facebook|instagram|other=" + $("#social").val() + "\n";
+  if ($("#opening_hours").val()) note_body += "⚠️ opening_hours=" + $("#opening_hours").val() + "\n";
+  if ($("#wheel").val()) note_body += "wheelchair=" + $("#wheel").val() + "\n";
+  if ($("#categoryalt").val()) note_body += "description=" + $("#categoryalt").val() + "\n";
+  if (paymentIds) note_body += paymentIds.join("\n") + "\n";
+  if ($("input:checked[name=delivery-check]").val() && $("#delivery").val() != "") note_body += "⚠️ delivery=" + $("#delivery").val() + "\n"; else if ($("input:checked[name=delivery-check]").val() && $("#delivery").val() == "") note_body += "delivery=yes" + "\n"; else if($('#delivery-check').not(':indeterminate') == true) note_body += "delivery=no" + "\n";
+  if ($("#delivery_description").val()) note_body += "delivery:description=" + $("#delivery_description").val() + "\n";
+  if ($("input:checked[name=takeaway]").val() != "undefined") note_body += "takeaway=" + $("input:checked[name=takeaway]").val() + "\n";
+  if ($("#takeaway_description").val()) note_body += "takeaway:description=" + $("#takeaway_description").val() + "\n";
+  if ($("input:checked[name=delivery_covid]").val() === 'Y') note_body += "delivery:covid19=yes\n";
+  if ($("input:checked[name=takeaway_covid]").val() == "yes" || $("input:checked[name=takeaway_covid]").val() == "only") note_body += "takeaway:covid19=" + $("input:checked[name=takeaway_covid]").val() + "\n";
+  if ($("#delivery_covid_description").val() || $("#takeaway_covid_description").val()) note_body += "description:covid19=";
+  if ($("#delivery_covid_description").val()) note_body += $("#delivery_covid_description").val() + " ";
+  if ($("#takeaway_covid_description").val()) note_body += $("#takeaway_covid_description").val() + "\n";
+  return note_body;
 }
 
 $("#collect-data-done").click(function() {
-    location.hash = '#done';
+  location.hash = '#done';
 
-    var latlon = findme_marker.getLatLng(),
-        qwarg = {
-        lat: latlon.lat,
-        lon: latlon.lng,
-        text: getNoteBody()
+  var latlon = findme_marker.getLatLng(),
+    qwarg = {
+      lat: latlon.lat,
+      lon: latlon.lng,
+      text: getNoteBody()
     };
-
-    $.post('https://api.openstreetmap.org/api/0.6/notes.json', qwarg, function( data ) {
-        console.log( data );
-        var noteId=data['properties']['id'];
-        var link='https://www.openstreetmap.org/?note='+noteId+'#map=19/'+latlon.lat+'/'+latlon.lng+'&layers=N';
-          $("#linkcoords").append('<a href="'+link+'">'+link+'</a>');
-    });
+  // 🔴🔴🔴🔴🔴 TODO: CAMBIARE CON https://api.openstreetmap.org/api/0.6/notes.json 🔴🔴🔴🔴🔴
+  $.post('https://api06.dev.openstreetmap.org/api/0.6/notes.json', qwarg, function(data) {
+    console.log(data);
+    var noteId = data['properties']['id'];
+    var link = 'https://api06.dev.openstreetmap.org/?note=' + noteId + '#map=19/' + latlon.lat + '/' + latlon.lng + '&layers=N';
```

Comment:
idem.. vuoi fare una var globale?

Reply:
In che senso?

Reply:
Ah ora ho capito. Ho modificato anche qui. Come detto lo avevo fatto solo per effettuare prove. 😀